workers/unity/Packages/com.improbable.gdk.transformsynchronization/TransfromSynchronizationSystemHelper.cs

```diff
@@ -4,6 +4,8 @@ namespace Improbable.Gdk.TransformSynchronization
 {
     public static class TransformSynchronizationSystemHelper
     {
+        public const int MaxBufferSize = 4;
```

Comment:
This shouldn't really be here as it's not a system helper. This is more feature module config.